web/src/render/useNotificationsRenderer.ts

```diff
@@ -105,7 +106,9 @@ export const useNotificationsRenderer: UsePixiRenderer = ({
       refreshInterval: REFRESH_INTERVAL,
       isPaused: () => !initialized,
       onSuccess: (newNotifications) => {
-        if (newNotifications.length > 0) {
+        const { length } = newNotifications;
+        if (length > 0) {
+          trackAnalyticsEvent("new-notifications", { length });
```

Comment:
this is going to be very very noisy - what value is there in tracking this?

Reply:
knowing that there were generated notifications for the user. Is there any other way to know about notifications?

Reply:
I see, so you want to know if the user get's to the point of generating notifications. I think in this case we should just fire this event once as opposed to every time the user gets notifications.

Reply:
let me know what you think of the new implementation